html/index.html

```diff
@@ -124,18 +124,18 @@ <h1 class="flush-top centered inverse short">
 						<div class="button-toolbar button-toolbar-centered flush-bottom download-buttons">
 
 							<div class="button button-primary inverse" id="button-download-osx">
-								<a href="/releases/v0.22.79/darwin-amd64/force" style="color:#58488a;">	Download for Mac OS X
```

Comment:
These links should be auto-generated based on the what comes back from the github apis, even though they are coded here they *should* be getting replaced via JS.